tests/UI/pages/FO/FObasePage.ts

```diff
@@ -17,7 +17,7 @@ export default class FOBasePage extends CommonPage {
 
   private readonly breadCrumbLink: (link: string) => string;
 
-  private readonly cartProductsCount: string;
+  public readonly cartProductsCount: string;
```

Comment:
a selector should not be public :
- `protected` if it's used on child class
- if used on a test : we should create the function that on the appropriate page that can be used on the test and make them private of course.

The exception for now maybe is `goToSubMenu` on the `BoBasePage`, but should be fixed too (maybe later because all tests depends on it)

Reply:
Ofc... But it's not on the scope of this PR (Migration JS => TS).

But I add a todo on the roadmap (#30263 in Last Steps).